<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>981</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des Employes</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>291</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>fonction</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>cin_e</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_adresse">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="la_fonction">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cin_e">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>290</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>671</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>List des Employes</string>
      </property>
      <widget class="QTableView" name="tab_Employes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>641</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="le_cin_supp">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>450</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SupprimerButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>401</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier </string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>fonction</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN: </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_adresse_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="la_fonction_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cin_e_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonModification">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>tab 1</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
